.github/workflows/base-wrapper.yml

```diff
@@ -73,20 +61,54 @@ on:
         type: string
         description: 'The tag to use for build stats uploads of native tests (e.g. 22.3.0-dev-jdk17-prepatch-x)'
         default: "null"
+      os:
```

Comment:
It looks like `os` is actually a GHA runner. So perhaps call this `gh_image` or `runner_image` or some such?